### ISSUE TITLE
Visual artifact with example scene

### ISSUE THREAD

**Issue by brookman** [mosure/bevy_gaussian_splatting#33]
Hi. Thank you for creating a Bevy implementation of Gaussian Splatting! 🦀

I have tried running the example using this command: `cargo run --release -- scenes/icecream.gcloud`

It compiles and runs but the results looks off (splats too large) and some of the gaussians are flickering badly (even when the camera is not moving):
![image](https://github.com/mosure/bevy_gaussian_splatting/assets/1568303/48bcfa5c-f622-4900-8a53-6d76fc98e1fa)

System information (Dell Precision 7560 notebook with Windows 11):

```
[Display]
DirectX version:	12.0 
GPU processor:		NVIDIA RTX A3000 Laptop GPU
Driver version:		536.45
Driver Type:		DCH
Direct3D feature level:	12_1
CUDA Cores:		4096 
Resizable BAR		Yes
Dynamic Boost 2.0	Yes
WhisperMode 2.0		No
Advanced Optimus	No
Maximum Graphics Power	105 W
Core clock:		1560 MHz 
Memory data rate:	11.00 Gbps
Memory interface:	192-bit 
Memory bandwidth:	264.05 GB/s
Total available graphics memory:	38504 MB
Dedicated video memory:	6144 MB GDDR6
System video memory:	0 MB
Shared system memory:	32360 MB
Video BIOS version:	94.04.42.40.06
IRQ:			Not used
Bus:			PCI Express x8 Gen4
Device ID:		XXX
Part Number:		XXX

[Components]

nvui.dll		8.17.15.3645		NVIDIA User Experience Driver Component
nvxdplcy.dll		8.17.15.3645		NVIDIA User Experience Driver Component
nvxdbat.dll		8.17.15.3645		NVIDIA User Experience Driver Component
nvxdapix.dll		8.17.15.3645		NVIDIA User Experience Driver Component
NVCPL.DLL		8.17.15.3645		NVIDIA User Experience Driver Component
nvCplUIR.dll		8.1.940.0		NVIDIA Control Panel
nvCplUI.exe		8.1.940.0		NVIDIA Control Panel
nvWSSR.dll		31.0.15.3645		NVIDIA Workstation Server
nvWSS.dll		31.0.15.3645		NVIDIA Workstation Server
nvViTvSR.dll		31.0.15.3645		NVIDIA Video Server
nvViTvS.dll		31.0.15.3645		NVIDIA Video Server
nvLicensingS.dll		6.14.15.3645		NVIDIA Licensing Server
nvDevToolSR.dll		31.0.15.3645		NVIDIA Licensing Server
nvDevToolS.dll		31.0.15.3645		NVIDIA 3D Settings Server
nvDispSR.dll		31.0.15.3645		NVIDIA Display Server
nvDispS.dll		31.0.15.3645		NVIDIA Display Server
PhysX		09.21.0713		NVIDIA PhysX
NVCUDA64.DLL		31.0.15.3645		NVIDIA CUDA 12.2.101 driver
nvGameSR.dll		31.0.15.3645		NVIDIA 3D Settings Server
nvGameS.dll		31.0.15.3645		NVIDIA 3D Settings Server
```

**Comment by mosure** [comment]
bevy discord help thread: https://discord.com/channels/691052431525675048/1179642698752335942

**Comment by mosure** [comment]
resolved in [`v2.0.2`](https://github.com/mosure/bevy_gaussian_splatting/tree/v2.0.2)
